### PR TITLE
docs: Explain backwards compat in SDK use case

### PR DESCRIPTION
**Pull request by charlieegan3**

### CODE REVIEW

File: docs/content/v0-compatibility.md [hunk 1]
```diff
@@ -113,6 +113,37 @@ r := rego.New(
 
 ### v0.x compatibility mode in the OPA Go SDK
 
-{{< info >}}
-TODO
-{{< /info >}}
+In OPA 1.0, the recommended
+[SDK package](https://pkg.go.dev/github.com/open-policy-agent/opa/v1/sdk)
+import for most users is `github.com/open-policy-agent/opa/sdk/v1`.
```
Comment (ashutosh-narkar) [inline docs/content/v0-compatibility.md]: `github.com/open-policy-agent/opa/v1/sdk`

File: docs/content/v0-compatibility.md [hunk 2]
```diff
@@ -113,6 +113,37 @@ r := rego.New(
...
+```go
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/open-policy-agent/opa/sdk" // <-- import v0 sdk package
+	// "github.com/open-policy-agent/opa/sdk/v1" // <-- import v1 sdk package
```
Comment (ashutosh-narkar) [inline docs/content/v0-compatibility.md]: `github.com/open-policy-agent/opa/v1/sdk`

Comment (ashutosh-narkar) [reply docs/content/v0-compatibility.md]: Also did you intend to keep this commented line?

Comment (charlieegan3) [reply docs/content/v0-compatibility.md]: I did, but I think it's not needed so removed in 8a81e679ee543e5ec18ecaa972389088314ee3c3

File: docs/content/v0-compatibility.md (hunk 1 above)
Comment (charlieegan3) [reply docs/content/v0-compatibility.md]: Fixed in 8a81e679ee543e5ec18ecaa972389088314ee3c3

**Review by ashutosh-narkar — APPROVED** [review]

**Pull request merged**